packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -350,7 +350,7 @@ class RenderParagraph extends RenderBox with ContainerRenderObjectMixin<RenderBo
   List<InlineSpanSemanticsInformation>? _cachedCombinedSemanticsInfos;
 
   /// The text to display.
-  InlineSpan get text => _textPainter.text!;
+  InlineSpan get text => _textPainter.text ?? const TextSpan();
   set text(InlineSpan value) {
     switch (_textPainter.text!.compareTo(value)) {
```

Comment:
Can this line also crash, since it also uses a `!` on the `_textPainter.text` property (like the getter above)